src/sonic-config-engine/tests/sample_output/py2/bgpd_quagga.conf

```diff
@@ -23,19 +23,11 @@ router bgp 65100
   bgp graceful-restart restart-time 240
```

Comment:
It is hard to check this kind of changes. Is it possible to preserve the previous file order?

Reply:
I think this issue could be resolve in the template itself.